BUFR_TableD_en_12.csv

```diff
@@ -711,3 +711,61 @@ Category,CategoryOfSequences_en,FXY1,Title_en,SubTitle_en,FXY2,ElementName_en,El
+12,Single level report sequences (satellite data),312072,(OSI SAF EPS-SG SCA soil moisture winds),,011081,Model wind direction at 10 m,,,,Operational
+12,Single level report sequences (satellite data),312072,(OSI SAF EPS-SG SCA soil moisture winds),,020095,Ice probability,,,,Operational
+12,Single level report sequences (satellite data),312072,(OSI SAF EPS-SG SCA soil moisture winds),,020096,"Ice age (""A"" parameter)",,,,Operational
+12,Single level report sequences (satellite data),312072,(OSI SAF EPS-SG SCA soil moisture winds),,008041,Data significance,,,,Operational
+12,Single level report sequences (satellite data),312072,(OSI SAF EPS-SG SCA soil moisture winds),,002099,Polarization,,,,Operational
+12,Single level report sequences (satellite data),312072,(OSI SAF EPS-SG SCA soil moisture winds),,201130,Change data width,Increase data width by 2 bits,,,Operational
+12,Single level report sequences (satellite data),312072,(OSI SAF EPS-SG SCA soil moisture winds),,202129,Change scale,Increase scaling by 1,,,Operational
+12,Single level report sequences (satellite data),312072,(OSI SAF EPS-SG SCA soil moisture winds),,011012,Wind speed at 10 m,,,,Operational
+12,Single level report sequences (satellite data),312072,(OSI SAF EPS-SG SCA soil moisture winds),,202000,Change scale,Cancel,,,Operational
+12,Single level report sequences (satellite data),312072,(OSI SAF EPS-SG SCA soil moisture winds),,201000,Change data width,Cancel,,,Operational
+12,Single level report sequences (satellite data),312072,(OSI SAF EPS-SG SCA soil moisture winds),,002099,Polarization,,,,Operational
+12,Single level report sequences (satellite data),312072,(OSI SAF EPS-SG SCA soil moisture winds),,008041,Data significance,,,,Operational
+12,Single level report sequences (satellite data),312072,(OSI SAF EPS-SG SCA soil moisture winds),,011052,Formal uncertainty in wind speed,,,,Operational
+12,Single level report sequences (satellite data),312072,(OSI SAF EPS-SG SCA soil moisture winds),,011080,Number of views used for wind retrieval,,,,Operational
+12,Single level report sequences (satellite data),312072,(OSI SAF EPS-SG SCA soil moisture winds),,021155,Wind vector cell quality,,,,Operational
+12,Single level report sequences (satellite data),312072,(OSI SAF EPS-SG SCA soil moisture winds),,201133,Change data width,Increase data width by 5 bits,,,Operational
+12,Single level report sequences (satellite data),312072,(OSI SAF EPS-SG SCA soil moisture winds),,021101,Number of vector ambiguities,,,,Operational
+12,Single level report sequences (satellite data),312072,(OSI SAF EPS-SG SCA soil moisture winds),,021102,Index of selected wind vector,,,,Operational
+12,Single level report sequences (satellite data),312072,(OSI SAF EPS-SG SCA soil moisture winds),,201000,Change data width,Cancel,,,Operational
+12,Single level report sequences (satellite data),312072,(OSI SAF EPS-SG SCA soil moisture winds),,101000,Replicate 1 descriptor n times,,,,Operational
```

Comment:
```suggestion
12,Single level report sequences (satellite data),312072,(OSI SAF EPS-SG SCA soil moisture winds),,101000,Delayed replication of 1 descriptor,,,,Operational
```

Reply:
@antoinemerle I don't understand why you resolved this without committing or commenting...